code/parachain/runtime/picasso/src/migrations.rs

```diff
@@ -145,6 +145,13 @@ pub mod pablo_picasso_init_pools {
 					CurrencyId::PICA_USDT_LPT,
 					Permill::from_rational::<u32>(3, 1000),
 				),
+				PoolCreationInput::new_two_token_pool(
+					CurrencyId::PICA,
+					Permill::from_percent(50),
```

Comment:
just to be sure, open/seeding pool will not require 50/50 tokens? so PICA != KSM? just double checking.

Reply:
Me and @benluelo are ensuring this via `add_liquidity` in another task